assets/js/controllers/transactions.controller.js

```diff
@@ -74,7 +74,7 @@ function TransactionsCtrl ($scope, $q, $translate, $uibModal, Wallet, MyWallet,
   $scope.$on('$destroy', unsub);
 
   // Searching and filtering
-  if (window.innerWidth <= 768) {
+  if ($scope.$root.size.sm || $scope.$root.size.xs) {
```

Comment:
This covers any width less than 992px, intentional? You had 768 before, just making sure.